src/pages/AdminDashboard/AdminWithdrawal.tsx

```
@@ -1,11 +1,467 @@
+import { useEffect, useState } from "react";
+import { useLazyNameEnquiryQuery } from "../../slices/accountApiSlice";
+import { toast } from "react-toastify";
+import {
+	useAllWithdrawalsQuery,
+	useFundAsAdminMutation,
+	useLazyApproveWithdrawalQuery,
+	useLazyDeclineDepositsQuery,
```

ESLint

[error] 'useLazyDeclineDepositsQuery' is defined but never used.

(@typescript-eslint/no-unused-vars)